js_modules/dagster-ui/packages/ui-core/src/assets/auto-materialization/AutomaterializationTickDetailDialog.tsx

```diff
@@ -202,6 +203,9 @@ const AssetDetailRow = ({
       },
     },
   );
+  const {data} = queryResult;
+  useBlockTraceOnQueryResult(queryResult, 'AssetGroupAndLocationQuery');
```

Comment:
I wonder if there's a way to read the query name out of the queryResult / apollo context here so that we don't need to put the name into a string each time... I feel like these could fall out of sync

Reply:
edit: hrmm i guess not, there's no reference to the `query` in the queryResult. Lame!